doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,147 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given pairs of source and target control points, TPS is used to transform a space,
+which, in our case, is a 2D image.
+
+
+.. [1] Wikipedia, Thin plate spline
+       https://en.wikipedia.org/wiki/Thin_plate_spline
+
+.. [2] Weisstein, Eric W. "Biharmonic Equation." From MathWorld--A Wolfram Web Resource
+       https://mathworld.wolfram.com/BiharmonicEquation.html
+
+.. [3] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+       decomposition of deformations." IEEE Transactions on pattern analysis and
+       machine intelligence 11.6 (1989): 567–585.
+       https://user.engineering.uiowa.edu/~aip/papers/bookstein-89.pdf
+
+
+Image Deformation
+=================
+
+Image deformation implies displacing the pixels of an image relatively to one another.
+In this example, we deform the (2D) image of an astronaut by using thin-plate splines.
+In our image, we define 6 source and target points labelled "1-6": "1-4" in the image
+corners, "5" near the smile corner and "6" in an eye. At the corners, there are no
+displacements. The smile corner moved upward and the eye position moved down.
+
+Thin Plate Splines provides a very handy interpolator for image deformation.
```

Comment:
Either "TPS" (as introduced earlier) or "thin-plate splines;" I would suggest a sentence starting with "We use TPS to ..." (interpolate / compute the ...).